src/Keboola/DbExtractor/Logger/LineFormatter.php

```diff
@@ -4,20 +4,14 @@
 
 namespace Keboola\DbExtractor\Logger;
 
-use Keboola\Csv\CsvFile;
-
 class LineFormatter extends \Monolog\Formatter\LineFormatter
 {
     /**
-     * @param CsvFile|array $data
+     * @param array|string $data
      * @return array|string
      */
     protected function normalize($data)
     {
-        if ($data instanceof CsvFile) {
-            return "csv file: " . $data->getFilename();
-        } else {
-            return parent::normalize($data);
-        }
+        return parent::normalize($data);
```

Comment:
to se muze uplne zrusit ta trida, ne?

Reply:
Jo jasne, resil jsem to tak tady https://github.com/keboola/db-extractor-common/pull/89/files#diff-ca73662417adcb930809592c2602d39c. Chci to delat strukturovane v PRs.